### ISSUE TITLE
Version 2 release

### ISSUE THREAD

**Issue by urdeveloper** [mapbox/tiny-sdf#37]
Thanks for making this library available to the open source community.
I was wondering if there's any plan to release v2 (changes since last release) in near future.
Cheers!

**Comment by mourner** [comment]
Yes! Planning to release this week along with a few more optimizations.

**Comment by urdeveloper** [comment]
Thank you very much! Closing the issue.

**Comment by mourner** [comment]
https://github.com/mapbox/tiny-sdf/releases/tag/v2.0.0
